composer.json

```diff
@@ -15,7 +15,7 @@
     "ext-libxml": "*",
     "ext-spl": "*",
     "ampproject/amp-toolbox": "0.6.0",
-    "cweagans/composer-patches": "1.7.1",
+    "cweagans/composer-patches": "~1.0",
     "fasterimage/fasterimage": "1.5.0",
     "sabberworm/php-css-parser": "dev-master#bfdd976"
```

Comment:
Looks like composer is locking the requirements of `dev-master` first, then checking out to `bfdd976`, causing a `psr-4` namespace mismatch (`"Sabberworm\\CSS\\": "src/"` instead of `"Sabberworm\\CSS\\": "lib/Sabberworm/CSS/"`) and thus not being able to autoload the library correctly.